tut_rule/tut.bzl

```diff
@@ -23,7 +23,7 @@ def scala_tut_doc(**kw):
     name = tool,
     main_class = "io.bazel.rules_scala.tut_support.TutCompiler",
     deps = deps + [
-      "//src/scala/io/bazel/rules_scala/tut_support:tut_compiler_lib"
+      "@io_bazel_rules_scala//src/scala/io/bazel/rules_scala/tut_support:tut_compiler_lib"
```

Comment:
I think we tried this approach and realized this means that now every user has to use this exact name for the rules scala repository, doesn't it? If so do we have an alternative?